### ISSUE TITLE
NC14.0.1 & Activity 2.7.0: "Deprecated file found - Please use repair steps"

### ISSUE THREAD

**Issue by Sanookmakmak** [nextcloud/activity#298]
`occ app:check-code activity`

```
Deprecated file found: /var/www/html/apps/activity/appinfo/update.php - please use repair steps
App is compliant - awesome job!
```

What does this mean: "Please use repair steps"?
Thank you!

**Comment by GAS85** [comment]
Also, it also does not show anything since update to NC 14:
![default](https://user-images.githubusercontent.com/6813635/46571012-c3e54180-c96d-11e8-8d45-1c175072db6a.png)

**Comment by nickvergessen** [comment]
Please open a new issue for that with more details as requested by the issue template

**Comment by DarkknightAK** [comment]
still an issue?

**Comment by GAS85** [comment]
It works fine at least already in 16 and 17.
Also it looks good:
```
# sudo -u www-data php /var/www/nextcloud/occ app:check-code activity
App is compliant - awesome job!
```

**Comment by nickvergessen** [comment]
Fixed with #369
